dashboard/src/views/SiteOverview.vue

```diff
@@ -46,6 +46,12 @@ const trialEndsText = computed(() => {
 	return utils.methods.trialEndsInDaysText(props.site.trial_end_date);
 });
 
+const isTrialEnded = computed(() => {
```

Comment:
```suggestion
const hasTrialEnded = computed(() => {
```

Boolean variable names in past tense.